src/lib.rs

```diff
@@ -155,10 +161,30 @@ pub fn init_with_rng<T: Rng>(rng: &mut T) -> Result<(), Error> {
 }
 
 /// Produces a 256-bit crypto hash out of the provided `data`.
-pub fn hash(data: &[u8]) -> [u8; 32] {
+pub fn hash(data: &[u8]) -> [u8; HASH_BYTES] {
     hashing_impl::sha3_256(data)
 }
 
+/// Seed structure used to generate sign and encrypt keypairs deterministically.
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Clone)]
+pub struct Seed {
+    seed: sign::Seed,
```

Comment:
`seed.seed` feels strange. Calling it `inner` is better ? Also check [review comment here](https://github.com/maidsafe/safe_crypto/pull/26#discussion_r219439233) pls.

Reply:
Replied to this on the linked review comment. We'll leave the names as they are.